### DISCUSSION TITLE
Wirkungsgrad Hoymiles bei 51% while on battery

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#242] [General]
I want to share a strange behaviour I experienced:
**My Test-Setup:**
- VICTRON MPPT 150/45 with four Solar panels charging two 25,6V 100Ah PowerQueen LiPo Batteries.
- Hoymiles HM-1500, all four inputs connected to the batteries (at 25,6V).
At 100%, Hoymiles delivers only 903,5W (much less than the 1500W max)
Each String delivers around 240W at 25V and 9,74A (this is near the HM-1500 max of 11,5A per input) 
The HM-1500 MPPT area is 36-48V thus my Voltage is below the MPTT area (I intend to change the batteries configuration to increase the voltage to 50V and test whether the max 1500W can be achieved).

**The Problem:**
Tonight we had problems with the "Dynamic Power limiter". The Hoymiles was delivering much less than  necessary to achieve "Nulleinspeisung". We identified the reason as bellow:
- The four inputs were delivering the correct amount of energy (Watt) need to achieve zero consumption.
- The inverter was converting **only half** the energy (Watt) of the inputs - **Wirkungsgrad 51%** 👎 
We tried many things but the Wirkungsgrad remained at 51%, **even after** switching off "Dynamic Power limiter" and switching the Inverter limit to 100%!
Only after a Restart of the inverter things went back to normal and the inverter started again working at  **Wirkungsgrad 95%**.

I do not understand what happened but may someone had the same experience.
**Some important setup parameters**
- Enable Solar-Passthrough - was active (maybe it should not have been?)
- Battery drain strategy: empty when full (When the sun is shining, this setting enables the sychronization of the inverter limit with the current solar power of the Victron MPPT charger. This optimizes battery degradation and loses)
- **DC Voltage - Start threshold: 27V** This may have been source of the problem, the battery was delivering 26V at that time, we changed it later to 25V

**Comment by madmartin** [comment]
IMHO the answer is easy:
The battery voltage is too low to achieve a good efficiency.
You mention all relevant data: 25V * ~10A -> ~250W x4 -> ~1000W. All fine. And if the inverter does not work efficiently with a voltage of 25V - then you have a badly matching hardware combination. I would expect here a very high temperature on the inverter, and the output power being significantly lower than maximum may be a side effect because of a thermal protection.

Higher voltages (and lower currents) are (as a rule of thumb) more efficient.
So I would advise you to check if you can put the batteries in serial to get 48V - that IMHO depends of your VICTRON MPPT 150/45 model and what the manufacturer of your   PowerQueen LiPo Batteries says to serial configuration. You should have a balancer connected to the batteries, which makes sure that the load (voltage) on the batteries is kept equal. Without such a device (usually part of the BMS) batteries in serial become not equally loaded, what shortens the life of the batteries dramatically.

**Comment by Manos1966** [comment]
Many thanks for the useful input (mad)Martin.
- Balancer - good point, I will have to look at this, thanks!
- The VICTRON MPPT 150/45 does 12 / 24 / 48 V (Auto Select) so, I think this will be a problem :-( (48V is not enough to charge a 2x25.6 = 51.2V as I understand). Real pity, because the VICTRON is amazing. 
The 50V configuration is planned to happen soon (we are waiting for one component to start the test).
- In terms of efficiency I tend to disagree. All four inputs reached near maximum Amps, at 9.8A (HM-1500 has 11.5A max). 
So, for a Voltage of 25V not a lot more could be achieved anyway. 
The 95% efficiency I consider acceptable, I do not believe I have ever seen my Hoymiles doing over 97% under optimal conditions.
The Hoymiles HM-1000,1200,1500 MPTT area is between 27V - 48V depending on the model thus, 24V, 25V, 50V, 51V batteries will not be within the optimization area anyways.
I am already very happy that the Hoymiles itself regulates input to around 10Amp (and does not burn...).
![HM1000-1200-1500 Eingang](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/da47ea88-dad6-49a3-a920-f05936ab5f22)
![HM-1500 on 4 Input 25V Battery](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/c7b2447a-5935-4786-b4f1-88e88df0538c)


By the way, the reason I made this post is because somehow yesterday the HM-1500 got "stuck" to a 51% efficiency and we could not explain why it was happening. After RESTART the inverter worked normal again (efficiency 95%).
We were testing the behaviour of "Dynamic Power Limit" yesterday and could not understand the very poor performance. By accident we realised that it was the poor efficiency % of the inverter that was causing the poor performance of the "Dynamic Power Limit" function. Unfortunately I did not make a screenshot of the condition.
I do not believe it was anything that had to do with OpenDTUonBattery and it is not a problem since it has not been repeated so far but, maybe someone will experience such behaviour.

**Comment by Manos1966** [comment]
I will post it separately but I add it also here,  
the Test with two "Power Queen 25,6V 100Ah LiFePO4, Eingebautes 100A BMS" batteries in serial to get 2x25,6 = 51V 
![HM-1500 on 4 Input 51V Battery](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/87acbdc7-ee27-4699-83ed-c9064dbcf751)
The HM-1500 set to 100% delivers 1475Watt at 95% efficiency 👍 
each of the four inputs runs at 45V and approx. 9A
As I understand (I maybe wrong, as I know about Solar & Batteries as much as about nuclear submarines... ) due to the heavy load, the voltage of the batteries dropped from 51V down to 45V, **which is supposed to be "normal"**  

Also interesting, the VICTRON 150/45 has switched (automatically) to 48V and is charging the batteries without problem - **I thought** since the batteries should have 51V, charging them with 48V VICTRON would be  a problem - it does not look like it. Is it because of the BMS each battery has?
@madmartin regarding Balancer, somebody told me that since each battery has its own BMS, there is no need to add an extra VICTRON BALANCER, maybe you know whether this assumption is correct.

**Comment by madmartin** [comment]
One remark to the victron charger - I own a 100/20 and the manual states that the battery voltage autodetection is done only ONCE on first "see" of the battery or after a reset to factory defaults. After this autodetect, the battery voltage is only changed  thriiugh the bluetooth app. In early firmware releases there was a "full" autodetection, which lead to mis-detections for example an completely empty 24V battery with a remaining voltage of 6V was detected (when disconnected and reconnected) as 12V battery. To avoid this effect, the voltage autodetection was changed to only happen once, and the manual strongly recommends to double-check this in the app.

You can change the voltage levels for the three load phases in the expert mode of the victron app, and you definitely should double check the voltages. Look into your battery documentation for maximum charge voltage and compare this (after multiplying x2) with the charger settings. Do not overcharge the batteries, they dont like that, and never discharge too deep, this kills accu cells shortly.

Another word about cell balancers in batteries. When you want a really long life of cells connected in serial, you need an electronic that checks, compares and equalizes the voltage of the cells.
If you have two batteries of 7 cells (assume 7 * 3,45V = 24,15V) with an integrated BMS, each group of these 7 cells is kept in equal load.
Imagine these two batteries have a small difference in capacity of some percent. Lets say you start with both batteries equally charged. Then discharge and recharge some dozen times. The battery with the lower capacity will be always a little overcharged, while the battery with the higher capacity will be more and more undercharged. This gets worse with every load cycle, and you should see you need to prevent this (the lower capacity battery will die soon) - with a balancing device (I dont know the exact word for it). This is not prevented by the two battery-internal BMS, it just not possible.

**Reply by Manos1966** [reply]
Many thanks for the explanations Martin, ! 
I will double check and look also for an external Balancer 👍

**Comment by Manos1966** [comment]
Just adding to the above, VICTRON MPPT 150/45 does 12 / 24 / 48 V, but as the picture shows, delivers 54,34V at 21,50A, enough to charge the 51V battery (my respect for VICTRON increased tremendously).
![Leistung VICTRON](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/1dc844d9-b86b-4e8d-9a80-6dcd67c3f9b1)
